.travis.yml

```diff
@@ -46,6 +46,22 @@ before_install:
   - mongod --version
   - mongo-orchestration --version
   - export MO_PATH=`python -c 'import mongo_orchestration; from os import path; print(path.dirname(mongo_orchestration.__file__));'`
+  - |
+    # tpecl is a helper to compile and cache php extensions
+      tpecl () {
+          local ext_name=ext-mongodb
+          local ext_so=mongodb.so
```

Comment:
I'd suggest we just copy the `tpecl()` definition from https://github.com/symfony/symfony/blob/32cc2e06a326b1dd888c02e1cc48988a14cb9a65/.travis.yml#L101 completely and parametrize the extension name and merge when invoking the `tpecl` function.